src/tasks/env.mk

```diff
@@ -2,7 +2,13 @@
 # Environment variables come from https://github.com/Financial-Times/next-vault-sync
 .env: ## .env: Downloads environment variables from Vault
 	@if [[ -z "$(shell command -v vault)" ]]; then echo "Error: You don't have Vault installed. Follow the guide at https://github.com/Financial-Times/vault/wiki/Getting-Started-With-Vault"; exit 1; fi
-	@if [[ -z "$(shell find ~/.vault-token -mmin -480)" ]]; then echo 'Error: You are not logged into Vault. Try vault login --method github token=$$VAULT_AUTH_GITHUB_TOKEN'; exit 1; fi
+	@if [  -z "$(shell find ~/.vault-token -mmin -480)" ]; then \
+		if [ -z "$(VAULT_AUTH_GITHUB_TOKEN)" ]; then \
+			echo "Error: VAULT_AUTH_GITHUB_TOKEN variable is not set."; exit 1; \
```

Comment:
could this also link to the "Getting Started" guide like the check for Vault being installed above?